src/compiler/crystal/command.cr

```diff
@@ -408,6 +408,13 @@ class Crystal::Command
         end
         opts.on("--static", "Link statically") do
           compiler.static = true
+          # https://developer.apple.com/library/content/qa/qa1118/_index.html
```

Comment:
Since the message to the terminal contains the link to Apple's website do you need this comment?